.github/workflows/distribute.yml

```diff
@@ -0,0 +1,111 @@
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Set up Ruby 2.7
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+
+      - name: Install bundler for ruby package management
+        run: gem install bundler -v "$(grep -A 1 "BUNDLED WITH" ./Gemfile.lock | tail -n 1)"
+        working-directory: ./android
+
+      - name: Install dotenv
+        run: gem install dotenv
+        working-directory: ./android
+        
+      - name: Cache ruby gems
+        uses: actions/cache@v1
+        id: ruby_cache
+        with:
+          path: android/vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/android/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Install Ruby Dependencies
+        if: steps.ruby_cache.outputs.cache-hit != 'true'
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+        working-directory: ./android
+        
+      ## Set up MN
+      - name: Setup MN env
+        run: ./bin/set_ha.sh mn ${{ secrets.ACCESS_TOKEN }}
+
+      - name: Save signing keys
+        run: ./bin/save_keystore.sh ${{ secrets.KEYSTORE_MN }}
+
+      - name: Increment version code
+        uses: maierj/fastlane-action@v1.4.0
+        with:
+          lane: 'release_increment_version_code'
+          subdirectory: 'android'
+          bundle-install-path: 'vendor/bundle'
+
+      - name: Build android release APK
+        uses: maierj/fastlane-action@v1.4.0
+        with:
+          lane: 'release_apk'
+          subdirectory: 'android'
+          bundle-install-path: 'vendor/bundle'
+          options: '{ "keyStorePassword" : "${{ secrets.KEYSTORE_PASSWORD_MN }}", "keyAlias" : "${{ secrets.KEY_ALIAS }}", "keyPassword" : "${{ secrets.KEY_PASSWORD_MN }}"}'
+
+      - name: Distribute APK
+        uses: maierj/fastlane-action@v1.4.0
+        with:
+          lane: 'android_alpha_apk'
+          subdirectory: 'android'
+          bundle-install-path: 'vendor/bundle'
+          options: '{ "apiToken": "${{ secrets.APPCENTER_TOKEN }}", "appName" : "packcheck-bt-mn" }'
+
+      # - name: Build android release AAB
+      #   uses: maierj/fastlane-action@v1.4.0
+      #   with:
+      #     lane: 'release_aab'
+      #     subdirectory: 'android'
+      #     bundle-install-path: 'vendor/bundle'
+      #     options: '{ "keyStorePassword" : "${{ secrets.KEYSTORE_PASSWORD_MN }}", "keyAlias" : "${{ secrets.KEY_ALIAS }}", "keyPassword" : "${{ secrets.KEY_PASSWORD_MN }}"}'          
+
+      # - name: Distribute AAB
+      #   uses: maierj/fastlane-action@v1.4.0
+      #   with:
+      #     lane: 'android_alpha_aab'
+      #     subdirectory: 'android'
+      #     bundle-install-path: 'vendor/bundle'
+      #     options: '{ "apiToken": "${{ secrets.APPCENTER_TOKEN }}", "appName" : "packcheck-bt-mn" }'
```

Comment:
We can't upload AAB files to AppCenter https://github.com/microsoft/appcenter/issues/955.
We should discuss which platform we can use to host those AAB files (they are required by Play Store).

Some options are:
- Google Drive (we need to set up https://github.com/google/skicka)
- Slack (someone with admin permissions will have to set up a bot and store the token as Github Secret)
- Github Actions Artifacts (those artifacts are public, maybe we can set up a hook that stores them in a private repository)

Reply:
We plan to move the distribution logic to Bitrise and store private artifacts there
https://trello.com/c/4jAoEC7f/276-set-up-a-ci-deployment-pipeline